webapp/cas-server-webapp-config/src/main/java/org/apereo/cas/web/security/CasWebSecurityConfigurerAdapter.java

```diff
@@ -294,6 +295,7 @@ private void configureEndpointAccessByAuthority(final ExpressionUrlAuthorization
             .httpBasic();
     }
 
+    @SuppressFBWarnings("BAS_BLOATED_ASSIGNMENT_SCOPE")
```

Comment:
I think the normal `SuppressFBWarnings` annotation can also be used, yes?  It'd be best to not have to tie into FB's API. Also, could the issue simply be "fixed"?

Reply:
Do you mean the `SupressWarnings` annotation rather than `SuppressFBWarnings`? I can fix this one fairly easily. There is a another one I did in a method with a few ternary operators that might be a little uglier to fix (doable but at at some cost to readability) but I will revisit that one too.